hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -542,30 +584,38 @@ static QString* psh_convert(struct PixelShader *ps)
     /* Window Clipping */
     QString *clip = qstring_new();
     if (ps->state.window_clip_count != 0) {
-        qstring_append_fmt(preflight, "uniform ivec4 clipRegion[%d];\n",
+        qstring_append_fmt(preflight,
+                           "uniform ivec4 clipRegion[%d];\n",
                            ps->state.window_clip_count);
-        qstring_append_fmt(clip, "/*  Window-clip (%s) */\n",
-                           ps->state.window_clip_exclusive ?
-                               "Exclusive" : "Inclusive");
+        qstring_append_fmt(clip,
+                           "/*  Window-clip (%s) */\n",
+                           ps->state.window_clip_exclusive ? "Exclusive" :
+                                                             "Inclusive");
```

Comment:
Here, (and in surrounding code) I feel the old code benefitted readability, as it was easier to spot the GLSL.

I'm not feeling very strongly about this either. So I'd tolerate this.